### DISCUSSION TITLE
Change Detection Colab notebook

### DISCUSSION THREAD

**Discussion by MarkoTukiainen** [azavea/raster-vision#2114] [Q&A] [answered]
Hello, 

Is there a way to access the Change Detection Colab notebook in the blog post about Change Detection, located at  https://www.azavea.com/blog/2022/04/18/change-detection-with-raster-vision/ since the link seems to be broken? Or is there similar sample code available somewhere else?

**Comment by AdeelH** [comment, marked as answer]
Hi, sorry about that! Looks like that file got lost. I've re-uploaded it here: https://colab.research.google.com/drive/1lZiyZhjgiLMK29Hk5rIBQv1TEL6_WNjV.

I should point out that this tutorial is for an older version of Raster Vision and has not been updated since. Please check out the [latest docs](https://docs.rastervision.io/en/0.30/usage/tutorials/index.html) and the [migration guides](https://docs.rastervision.io/en/0.30/migration/index.html) if you want to make the code compatible with the latest version.
